unit_tests/test_aggregator.py

```diff
@@ -99,137 +79,125 @@ def fixture_aggregator_from_env(mocker):
     return Aggregator(aggregator_name=test_agg["name"])
 
 
-#####################################
-# DEF TESTS FOR CLASSES CONSTRUCTOR #
-
 class TestConstructor:
-    """Includes methods which test constructor."""
+    """Test the constructor use cases of the aggregator."""
 
     @staticmethod
-    def test_constructor__simulation_id_explicit(aggregator_explicit):
+    def test_constructor_simulation_id_explicit(aggregator_explicit):
```

Comment:
I'd like to inquire here about the main reason why you suggest not using this double underscore.

IMO it helps to identify in an easier manner which use case is being tested. I'm not sure however if i'm violating any PEP rule or smthn.

Reply:
@ERafaelMartinez We don't have a written rule on this in our guidelines but all of our test methods so far use a single underscore between words, if we're to change this it would be awesome if you can raise this topic in the backend channel so others can give their opinion

Reply:
Actually it seems to be already written https://gridsingularity.atlassian.net/wiki/spaces/D3A/pages/2406449155/Naming+and+Style+conventions?focusedCommentId=3147104267#Naming-conventions